manifests/06_clusterrole.yaml

```diff
@@ -261,4 +261,12 @@ rules:
   resources:
     - configmaps
   verbs:
-    - '*'
\ No newline at end of file
+    - '*'
+- apiGroups:
+    - csi.storage.k8s.io
+  resources:
+    - csinodeinfos
+  verbs:
+    - get
+    - list
+    - watch
```

Comment:
Do you need `update` access as well?

Reply:
Ambika tried the current branch on her IPI cluster and it is working as expected. It is not needed now.